src/pytorch_lightning/strategies/ipu.py

```diff
@@ -269,7 +269,7 @@ def to_tensor(x):
         args = apply_to_collection(args, dtype=(int, float), function=to_tensor)
         return args
 
-    def batch_to_device(self, batch: Any, device: Optional[torch.device] = None, dataloader_idx: int = 0) -> Any:
+    def batch_to_device(self, batch: Any, *_: Any, **__: Any) -> Any:
```

Comment:
can we still keep the signature as is? Even though these args may not be used, they should be part of the signature IMO as this really improves readability

Reply:
reverted!